packages/web3-utils/src/hash.ts

```diff
@@ -16,33 +16,33 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 import {
-	InvalidStringError,
-	InvalidBooleanError,
 	InvalidAddressError,
-	InvalidSizeError,
+	InvalidBooleanError,
+	InvalidBytesError,
 	InvalidLargeValueError,
+	InvalidSizeError,
+	InvalidStringError,
 	InvalidUnsignedIntegerError,
-	InvalidBytesError,
 } from 'web3-errors';
 import { keccak256 } from 'ethereum-cryptography/keccak';
 import { isAddress, isHexStrict, isNullish } from 'web3-validator';
 import {
+	Bytes,
+	EncodingTypes,
 	Numbers,
+	Sha3Input,
 	TypedObject,
 	TypedObjectAbbreviated,
-	EncodingTypes,
-	Bytes,
-	Sha3Input,
```

Comment:
pls avoid such changes and only white space changes in files , specially in large PRs.

Reply:
done